global_install_scripts/install.bash

```diff
@@ -228,6 +228,7 @@ function run() {
 		if [ -f $FILE_PATH ] && grep -q "TALISMAN_HOME" $FILE_PATH; then
 			if ! grep -q ">>> talisman >>>" $FILE_PATH; then
 				sed -i'-talisman.bak' '/TALISMAN_HOME/d' $FILE_PATH
+				echo "\n" >>${ENV_FILE}
```

Comment:
Please change this and all subsequent occurrences to `echo -e "\n"`

Reply:
have changed. thanks